docs/guides/compose.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Compose
+---
+
+The Tailcall CLI's `compose` command facilitates the merging of configuration files and their conversion between different formats using the `--format` option.
```

Comment:
Intro should talk about the fact that tailcall support three input formats in configurations.